meson.build

```diff
@@ -34,11 +34,12 @@ config_piper.set('pkgdatadir', pkgdatadir)
 config_piper.set('localedir', localedir)
 config_piper.set('gtk_major_version', gtk_major_version)
 config_piper.set('gtk_minor_version', gtk_minor_version)
+config_piper.set('devel', '')
```

Comment:
I can imagine what this does, but please add the failure message to the commit message, it makes it much easier to a) understand what's going on and b) identify if any other future failures are caused by separate issues or by an insufficient patch.